### DISCUSSION TITLE
ClientId displayed on browser's console and sensitivity of this info

### DISCUSSION THREAD

**Discussion by flbouchut** [Web3Auth/Web3Auth#837] [Help]
Hello there, 

Is the clientId a sensitive information and how to hide it ? It is displayed by default by Sentry on the browser's console when login using email authentication. 

<img width="616" alt="image" src="https://user-images.githubusercontent.com/38558474/200596142-8a086971-e256-438e-b4dd-2f927c5f5452.png">

Thanks,

**Comment by shahbaz17** [comment]
Hey @flbouchut 

The `clientId` is not a piece of sensitive information. It is mapped to your domain that you whitelist on the [Web3Auth Dashboard](https://dashboard.web3auth.io). And only that domain(s) will be allowed to use this `clientId`.

**Reply by flbouchut** [reply]
All right, thank you for your quick answer !
